src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import org.hibernate.SessionFactory;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDao dao;
+
+    public ActorServiceImpl(SessionFactory factory) {
+        dao = new ActorDaoImpl(factory);
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return dao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        return dao.get(id).orElseThrow();
```

Comment:
Well, not just calling this method, but throwing an exception if the object cannot be obtained. :)